.script/tests/idChangeValidatorTest/SkipIdValidationsTemplates.json

```diff
@@ -234,5 +234,19 @@
   "4f971586-9624-429b-80c4-3c0c940c1962",
   "85695071-6425-4ebf-a2f9-e7a827569848",
   "9bbf2a02-a20d-4eaa-ab74-3b60cfe6f3d3",
-  "939d1daa-9ee5-43ae-ae96-12c30c41e528"
+  "939d1daa-9ee5-43ae-ae96-12c30c41e528",
+  "94f9ffe1-7d9a-4fe9-8949-93322f090d04",
+  "a0dde092-d143-449d-8cbc-b5e4b5b261b8",
+  "ea60f4af-d37c-442f-a83e-6d9f9c5d0d2d",
+  "e027ec78-39eb-4cdc-8fda-ed41c8cd3d4f",
+  "c1c2d0a6-aab4-4f6d-8786-8bb0d37eba03",
+  "ad6882a8-7749-471e-b7d6-6c7e42a8eca3",
+  "c4f6f1e2-2cbb-4e48-90f4-cd8c11b45483",
+  "8630a18a-60a1-428a-ae63-59b5129a1c72",
+  "642f20de-b4cb-4cf3-8879-6073ef504fcd",
+  "3821c666-f600-49dd-ac2a-19c61bcae619",
+  "0f28250d-5a01-4be3-83f0-6e16199652d0",
+  "2458128d-a3c7-4af3-9476-f8a9bbb24f49",
+  "84c1c385-c3b7-4885-a92f-285e8411be2b",
```

Comment:
All of the newly added Guids are not referring to any file in Azure Sentinel project. Please add moved files in there original location with new ids in it. Refer this guidelines. https://microsoftapc.sharepoint.com/:w:/t/SentinelCATTeam/EVRk6UGiVPdMiMeap-ihhlsBIeMSxzNyJMcD3VmfRD9IMw?e=t51F7V&CID=5EA794BE-8633-4821-821A-A3034DBFA79B

Reply:
there was merge conflicts please review now